setup.py

```diff
@@ -27,7 +27,7 @@
         'crate>=0.16',
         'argh',
         'tqdm',
-        'Faker==0.9.1',
+        'Faker==4.1.0',
```

Comment:
Instead of pinning `faker` to a specific version, I could also try to find a common base and add a version range, something like `Faker>=1.0<5.0`

Reply:
Maybe you could go with `>=4.0.0,<5.0`. Looks like they're using sem-ver, so that should be fine.

Reply:
Thanks. Changed.